logsearchapi/main.go

```diff
@@ -32,9 +32,10 @@ import (
 )
 
 const (
+	// QueryAuthTokenEnv environment variable
+	QueryAuthTokenEnv = "LOGSEARCH_QUERY_AUTH_TOKEN"
 	pgConnStrEnv      = "LOGSEARCH_PG_CONN_STR"
 	auditAuthTokenEnv = "LOGSEARCH_AUDIT_AUTH_TOKEN"
-	queryAuthTokenEnv = "LOGSEARCH_QUERY_AUTH_TOKEN"
 	diskCapacityEnv   = "LOGSEARCH_DISK_CAPACITY_GB"
 )
 
```

Comment:
Make sure all ENVs are exported, in-case you wish to modify each of them @cesnietor - also its generally advised never to import a `main` package, if needed please move the relevant constants to a package inside `logsearchapi/api/`

Reply:
I've added them to the current server package. let me know if it makes sense.